randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -742,7 +742,9 @@ Extra - asset_id: collision_camera_011
               Space Jump and Movement (Intermediate) and Lay Bomb
           Any of the following:
               # A way to climb into the tunnel, after the bomb blocks are gone
-              Infinite Bomb Jump (Intermediate) or Walljump (Advanced) or Use Spin Boost
+              Walljump (Advanced) or Use Spin Boost
+              Infinite Bomb Jump (Intermediate) and Lay Cross Bomb
+              Infinite Bomb Jump (Advanced) and Lay Bomb
```

Comment:
Technically, this wouldnt need an IBJ with Cross Bombs, if it's even possible to do with cross. I would take out the cross one until tested if cross can do it.

Reply:
I took out the IBJ with Cross, but I found out that the strat for Normal Bombs works with just Spin Boost. Not sure how one would make use of Space Jump.